projects/core/src/occ/adapters/asm/occ-asm.adapter.spec.ts

```diff
@@ -47,7 +51,11 @@ class MockBaseSiteService {
 }
 
 class MockOccEndpointsService {
```

Comment:
Would be good to add `implements Partial<OccEndpointsService>` to make future refactoring easier.